modular_causticcove/code/modules/cargo/packsrogue/merchant/weapons/merch_weapons_foreign.dm

```diff
@@ -0,0 +1,21 @@
+/*
+/datum/supply_pack/rogue/merc_weapons/arquebus
+	name = "Grenzelhoft Arquebus Rifle"
+	cost = 150
+	contains = list(/obj/item/gun/ballistic/arquebus)
+
+/datum/supply_pack/rogue/merc_weapons/arquebus_pistol
+	name = "Grenzelhoft Arquebus Pistol"
+	cost = 120
+	contains = list(/obj/item/gun/ballistic/arquebus_pistol)
+*/ // This is too big of a change right now, but it's here if anyone in the future wants to use it.
+
+/datum/supply_pack/rogue/merc_weapons/arquebus_powderflask
+	name = "Grenzelhoft Arquebus Powderflask"
+	cost = 50
```

Comment:
This is the thing I am least confident about. Is this price good or should it be higher?

Whether or not the engineer can make powderflasks hinges entirely on if either
1) They are minmaxing and can farm infernal ash by summoning things to kill and also aren't being hounded by 20 players who all want them to do smithing.
2) A mage is also online and willing to give/sell infernal ash instead of farming fairy dust to break the economy.
3) Someone has visited Mt. Decap and thought to bring back some ash.

So if you lose your powderflask, you're fucked.